translations.yml

```diff
@@ -1090,18 +1090,23 @@
 
 - en: 'Editors:'   # gender-neutral, plural. Precedes list of more than one person who edited the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Edition'.
   fr: 'Rédaction :'
+  de: 'Redaktion:'
 
 - en: 'Update Editor:'   # gender-neutral, singular. Precedes list of one person who updates the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Update Edition'.
   fr: 'Rédaction des mises à jour:'
+  de: 'Redaktion für Aktualisierungen'
 
 - en: 'Status:' # followed by the last updated date or
   fr: 'Statut :'
+  de: 'Status'
 
 - en: 'Updated' #  precedes a date
   fr: 'Mise à jour le'
+  de: 'Aktualisiert'
 
 - en: 'first published' #  precedes a date
   fr: 'première publication'
+  de: 'erstmals veröffentlicht'
```

Comment:
Depending on context, might read better if followed by “ am”

Reply:
I.e., `de: 'erstmals veröffentlicht am'`

Reply:
Context is this in English:

> Status: Updated 13 April 2017 (first published May 2015)

Current proposed translations would read as follows:

> Status Aktualisiert 13 April 2017 (erstmals veröffentlicht Mai 2015)

Reply:
Thanks!

This needs more dissecting.

1. The “Status” text does miss a colon as mentioned earlier.
2. For the dates, might both “full” (“13 April 2017”) and partial dates (“Mai 2015”) be used?
  - If only full (and example is inaccurate), “Aktualisiert am” and “erstmals veröffentlicht am” should read better
  - If either, omit “am” as originally translated (“Aktualisiert”, “erstmals veröffentlicht”) – doesn’t read as nice, but works best in this case of date format ambiguity
3. The full date should include a period after the day: “13. April 2017”

Reply:
> For the dates, might both “full” (“13 April 2017”) and partial dates (“Mai 2015”) be used?

- Only full for the last updated date.
- Full or partial (mainly partial) for the first published date.

> The full date should include a period after the day: “13. April 2017”

Thanks for this clarification! It will not be the case when we publish this translation (the code does not support different date formats depending on the language), but it is important to know for future improvements.

Reply:
> I assumed that for the sake of brevity, the shorter version is preferred over the correct one

I have the same assumption. 

Feel free to choose your preferred version according to the given context. This could easily be changed in the future if needed.